usr/share/rear/finalize/Linux-ppc64/540_check_lilo_path.sh

```diff
@@ -0,0 +1,28 @@
+# THIS SCRIPT CONTAINS PPC64 SPECIFIC
+#################################################################
+# The purpose of this script is to check if the "part" variable in /etc/lilo.conf
+# is an existing disk partition on the local system. If not replace it with the PReP
+# partition path.
+#
+# This script must be run before 610_intall_lilo.sh and 550_rebuild_initramfs.sh
+#################################################################
+
+# skip if lilo conf is not found
+test -f $TARGET_FS_ROOT/etc/lilo.conf || return
+
+# Find PPC PReP Boot partitions
+part=$( awk -F '=' '/^boot/ {print $2}' $TARGET_FS_ROOT/etc/lilo.conf )
```

Comment:
Only an addedum FYI regarding
"get data located in a specific field/column":

Usually I use an artificial bash array to get a specific word
inside a line of output words, for example:
<pre>
fstab_rootfs_entry=( $( grep -v '^#' /etc/fstab | grep ' / ' ) )
rootfs=${fstab_rootfs_entry[2]}
</pre>
Of course this "just works" only when fields/columns
are separated via $IFS.